editor/editor_properties.cpp

```diff
@@ -3363,7 +3363,7 @@ void EditorPropertyResource::update_property() {
 			sub_inspector = nullptr;
 
 			if (opened_editor) {
-				EditorNode::get_singleton()->hide_unused_editors();
+				EditorNode::get_singleton()->hide_unused_editors(this);
```

Comment:
This might cause problems if the property had a sub-inspector editing another Resource. Though not sure if it can happen (outside of custom plugins at least).

Reply:
From the logic of `edit_item()`, a plugin can only edit one object at a time. However, I am not sure what happens when the property is collapsed, so it may be safer to modify it back.